src/UITests/AIWinSession.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using Axe.Windows.Automation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using OpenQA.Selenium.Remote;
+
+namespace UITests
+{
+    [TestClass]
+    public class AIWinSession
+    {
+        protected const string WindowsApplicationDriverUrl = "http://127.0.0.1:4723";
+
+        protected static WindowsDriver<WindowsElement> session;
+        protected static int processId;
+
+        public static void Setup(TestContext context)
+        {
+            // Launch a new instance of application
+            if (session == null)
+            {
+                // Create a new session to launch application
```

Comment:
The mere existence of this comment indicates that all the code below it could be broken out into a separate, private function. The function would have a similar name to the text of the comment.